os/net/ipv6/uip-ds6-nbr.c

```diff
@@ -113,10 +170,92 @@ uip_ds6_nbr_add(const uip_ipaddr_t *ipaddr, const uip_lladdr_t *lladdr,
   }
 }
 
+#if UIP_DS6_NBR_MULTI_IPV6_ADDRS
+/*---------------------------------------------------------------------------*/
+static void
+add_uip_ds6_nbr_to_nbr_entry(uip_ds6_nbr_t *nbr,
```

Comment:
Some doxygen would be really nice for all these new functions :) Their behavior is not very obvious.